app/src/components/tasks/TaskList.tsx

```diff
@@ -80,6 +80,7 @@ export default function TaskList(props: TaskListProps) {
             <TaskCompletionBody
```

Comment:
Ideally I'd like to refactor `TaskCompletionBody` to just take in the entire `selectedTask` now that we're passing in so many of its params but in the interest of time decided not to do it as part of this PR

Reply:
Totally makes sense! Not all of this is the cleanest coding we've ever done 😬